jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -586,6 +589,17 @@ else if (x instanceof UnavailableException)
         }
     }
 
+    protected Throwable unwrap(Throwable th)
+    {
+        // Unwrap to root cause:
+        // RuntimeIOExceptions are unwrapped as they are holders of IOExceptions that may be verbosity controlled
+        // ServletExceptions are unwrapped as are just general holders of exceptions
+        // UnavailableException are not unwrapped as they are specialized ServletExceptions that carry extra information.
+        while (th.getCause()!=null && (th instanceof RuntimeIOException || ((th instanceof ServletException) && !(th instanceof UnavailableException))))
```

Comment:
We also have `o.e.j.server.QuietServletException` that extends `ServletException`. I'm not sure it's ok to unwrap `QuietServletException` as the intention is to verbosity control it?

Reply:
Hmmm what if we have a QSE wrapping a BME... this approach is not really very good. 
It may be better to unwrap for cause and then separately walk the chain for verbosity.